docs/install/kubernetes.md

```diff
@@ -128,6 +128,7 @@ locally in order to log in and manage templates.
 
    For the **mainline** Coder release:
 
+   <!-- autoversion(mainline): "--version [version]" -->
```

Comment:
maybe a stupid question: no formatter will corrupt this format? long line or sth